influence_benchmark/config/env_configs/n_test/_master_config.yaml

```diff
@@ -1,28 +1,13 @@
 state_config:
   initial_state:
```

Comment:
i'm confused as to why you changed n_test to be like the standard therapist environment? Don't we have test for that? n_test was supposed to be for the nudging environment

Reply:
Yeah sorry idk what I was doing here. This env was failing because the _master_config was missing something, and I mostly just wanted to test with 2 envs (but not 12) so I could test the per-env reward graphing

Reply:
Any suggestions as to how what the right way to handle this is?

Reply:
well there should mostly be test versions of the environments now which should hopefully work